### ISSUE TITLE
Feature Request: Color Impairment Accessibility 

### ISSUE THREAD

**Issue by pushfoo** [DragonMoffon/lux-aenigma#3]
TL;DR: Since there are ~8 colors, we can map to patterns

@DigiDuncan was showing me the game as it is. Color-impairment modes immediately came to mind. In addition to color filter modes for partial color blindness, one idea I suggested was using patterns to enhance differences between the colors.

Digi made a mock-up in both Color and B&W modes:
| With Color | B&W* |
|---------------|----------|
| ![image](https://github.com/DragonMoffon/lux-aenigma/assets/36696816/0eb370f1-0864-4aea-81bd-69353b21ad6d) | ![image](https://github.com/DragonMoffon/lux-aenigma/assets/36696816/f7d875b1-ee3d-4994-9f95-a58ba869fe07) |

**\* There are multiple issues with this B&W conversion**

1. Most graphics programs do not have a mono conversion option which accurately replicates human perception's color mixing
    * **Note: Digi mentions that Adobe may have added one in PS at some point*
    * It may be worth implementing this as a shader
3. We'll need to research how to maximize visual contrast in the grayscale band
    * Contrast perception in grayscale may have been partially logarithimic instead of linear
    * We may need 10+ total shades to account for black, game values, and white for object / UI interactions?

**Comment by DragonMoffon** [comment]
Yup, this was in the plan, but good idea making an issue for it
